src/jetson/models/utils/box_utils.py

```diff
@@ -67,6 +67,30 @@ def jaccard(box_a:torch.Tensor, box_b:torch.Tensor):
     union = area_a + area_b - inter
     return inter / union  # [A,B]
 
+def matrix_iou(a, b):
+    """
+    return iou of a and b, numpy version for data augenmentation
+    """
```

Comment:
can you type hint this. Also maybe I'm sure this is a common function so I'm not gonna ask you to explain it but a link to something that does would be nice